From 4701d802d65948b738d3112387306621bdcd849f Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Fri, 4 Jun 2021 11:20:18 +0800
Subject: [PATCH 2/5] tweak build option for lat

Upstream-Status: Inappropriate [lat specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 debian/rules | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/debian/rules b/debian/rules
index 59848351..c5db0eac 100755
--- a/debian/rules
+++ b/debian/rules
@@ -1,5 +1,6 @@
 #!/usr/bin/make -f
 
+export DEB_CFLAGS_MAINT_APPEND = -Werror=implicit-function-declaration -Werror=int-conversion
 export DEB_BUILD_MAINT_OPTIONS = hardening=+all
 
 binaries := $(shell dh_listpackages)
@@ -11,20 +12,22 @@ override_dh_autoreconf:
 	env NOCONFIGURE=1 dh_autoreconf ./autogen.sh
 
 configure_options = \
+	GJS=some-nonexistent-gjs-binary \
 	--enable-installed-tests \
+	--libexecdir='$${prefix}/libexec' \
 	--with-avahi \
-	--with-dracut \
+	--without-dracut \
 	--with-grub2 \
 	--with-grub2-mkconfig-path=/usr/sbin/grub-mkconfig \
 	--with-libarchive \
 	--with-libsystemd \
 	--with-libmount \
-	--with-modern-grub \
 	--with-soup \
-	--with-selinux \
+	--without-selinux \
 	--with-systemdsystemgeneratordir=/lib/systemd/system-generators \
 	--with-systemdsystemunitdir=/lib/systemd/system \
 	--without-smack \
+	--with-builtin-grub2-mkconfig \
 	$(NULL)
 
 ifneq ($(filter libostree-doc,$(binaries)),)
-- 
2.34.1

