From acd22da55ceed711242e85c0b6167c7ac4053f0d Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Thu, 10 Jun 2021 11:28:32 +0000
Subject: [PATCH 4/5] set ostree osname for lat

Upstream-Status: Inappropriate [lat specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 debian/rules | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/rules b/debian/rules
index b689bdd..ab8d80f 100755
--- a/debian/rules
+++ b/debian/rules
@@ -9,6 +9,7 @@ binaries := $(shell dh_listpackages)
 	dh $@ --with=gir
 
 override_dh_autoreconf:
+	sed -i -e "s#@OSTREE_OSNAME@#${OSTREE_OSNAME}#g" ./src/libostree/ostree-impl-system-generator.c
 	env NOCONFIGURE=1 dh_autoreconf ./autogen.sh
 
 configure_options = \
-- 
2.34.1

