From 7e642bef1db4e0a39355401775ba15509a650d79 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Thu, 8 Apr 2021 17:38:41 +0800
Subject: [PATCH] make timezone support Hong Kong

Upstream-Status: Pending

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 src/ostree/parse-datetime.y | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ostree/parse-datetime.y b/src/ostree/parse-datetime.y
index e1ce3057..5db13128 100644
--- a/src/ostree/parse-datetime.y
+++ b/src/ostree/parse-datetime.y
@@ -793,6 +793,7 @@ static table const time_zone_table[] =
   { "CDT",      tDAYZONE, -HOUR ( 6) }, /* Central Daylight */
   { "MST",      tZONE,    -HOUR ( 7) }, /* Mountain Standard */
   { "MDT",      tDAYZONE, -HOUR ( 7) }, /* Mountain Daylight */
+  { "HKT",      tZONE,    -HOUR ( 8) }, /* Hong Kong Time */
   { "PST",      tZONE,    -HOUR ( 8) }, /* Pacific Standard */
   { "PDT",      tDAYZONE, -HOUR ( 8) }, /* Pacific Daylight */
   { "AKST",     tZONE,    -HOUR ( 9) }, /* Alaska Standard */
-- 
2.27.0

